server/src/main/java/org/apache/uniffle/server/ShuffleTaskManager.java

```diff
@@ -833,4 +853,47 @@ private void triggerFlush() {
       this.shuffleBufferManager.flushIfNecessary();
     }
   }
+
+  public List<Map.Entry<String, ShuffleTaskInfo>> calcTopNTotalDataSizeTaskInfo() {
```

Comment:
How about putting these calcTopN methods into `TopNShuffleDataSizeOfAppCalcTask`

Reply:
I original want to  encapsulate shuffleTaskInfos in ShuffleTaskManager.
If do as what you say, shuffleTaskInfos will be public through get method().

I will modify the code to your idea.

Reply:
I added test case  [TopNShuffleDataSizeOfAppCalcTaskTest.java](https://github.com/apache/incubator-uniffle/pull/1400/files#diff-708bd5b5ae683a06ec9b9c8f7a4506afe3ac2508bb7dbdd2d11feeb29fdf72d6)